components/omnibox/browser/BUILD.gn

```diff
@@ -2,31 +2,16 @@ source_set("browser") {
   # Only //components/omnibox/browser target can depend on this target
   # because this target expands //components/omnibox/browser implementation.
   visibility = [ "//components/omnibox/browser" ]
-  # Remove when https://github.com/brave/brave-browser/issues/10624 is resolved
-  check_includes = false
-
   sources = [
-    "brave_omnibox_client.cc",
-    "brave_omnibox_client.h",
     "constants.cc",
     "constants.h",
     "suggested_sites_match.cc",
     "suggested_sites_match.h",
-    "suggested_sites_provider.cc",
-    "suggested_sites_provider.h",
-    "suggested_sites_provider_data.cc",
-    "topsites_provider_data.cc",
-    "topsites_provider.cc",
-    "topsites_provider.h",
   ]
 
   deps = [
      "//base",
      "//components/omnibox/browser:buildflags",
```

Comment:
this dep doesn't appear to be used by any of the remaining sources, but as I mention above I'm not clear on why some have been left here

Reply:
yes, it's unnecessary because there is no use of ENABLE_VR in the directory.